profiler/src/ProfilerEngine/DeployResources/DDProf-SetEnv.bat

```diff
@@ -33,26 +33,26 @@ IF "%2"=="" (
     ECHO     Use the second command line parameter to specify the DD-Prof-DotNet Alpha Deploy Directory.
     ECHO     Using default.
 
-    SET DD_DOTNET_PROFILER_HOME=%~dp0
+    SET DD_DEPLOY_DIRECTORY=%~dp0
```

Comment:
Orrrr..... we could delete these files 😉

Reply:
I prefer to wait for Gregory to come back and decide what to do  ;^)